next.config.js

```diff
@@ -35,6 +35,10 @@ const nextConfig = {
         protocol: 'https',
         hostname: 'www.example.com',
       },
+      {
+        protocol: 'https',
+        hostname: 'images.unsplash.com'
+      }
```

Comment:
why ?

Reply:
did it for some testing locally, will raise a new pr after removing it.

Reply:
@VineeTagarwaL-code please check now.